src/FffGcodeWriter.cpp

```diff
@@ -399,6 +399,11 @@ void FffGcodeWriter::processStartingCode(const SliceDataStorage& storage, const
 
     gcode.writeComment("Generated with Cura_SteamEngine " VERSION);
 
+    if (gcode.getFlavor() == EGCodeFlavor::REPRAP)
```

Comment:
Please also move the `T0` which is already being outputted for `GRIFFIN` here.

Line 444: `gcode.writeCode("T0"); // Toolhead already assumed to be at T0, but writing it just to be safe...`
should be moved here as well, by extending the if condition above.